pong-sandbox/main.py

```diff
@@ -1,16 +1,46 @@
 import pygame
 
-from pong import Ball, Player
+from pong import Pong
 
-def start_game():
-    pass
+def start_game(window, game):
+
+    move_up, move_down = -1, 1
+    shouldRun = True
+    while shouldRun:
+        pygame.time.delay(200)
```

Comment:
Why delay is so high. 20 is ok right?

Reply:
20 ms might end up too high when we implement the speed boost functionality. But we can work on this!